godot-core/src/builtin/basis.rs

```diff
@@ -264,17 +264,22 @@ impl Basis {
         }
     }
 
-    fn to_euler_inner(major: f32, fst: Vec2, snd: Vec2, rest: Vec2) -> Vec3 {
+    fn to_euler_inner(
+        major: f32,
+        fst: glam::Vec2,
+        snd: glam::Vec2,
+        rest: glam::Vec2,
```

Comment:
These parameter names are not very clear. I assume `fst`/`snd` mean 1st/2nd? What about the others?

Reply:
I'm honestly not sure what to name these. But my idea for these were roughly:
There are two cases
Case one: where `-1 < major < 1`. so i named it major since it decides where to branch
Then in the equal case, we have one element be based off of `major`, and then we have `fst` and `snd` in order (though i realize now after some refactoring i've managed to flip those two around)

Reply:
the algorithm is based on [this](https://en.wikipedia.org/wiki/Euler_angles#Rotation_matrix) mapping, so im not sure what to name it entirely. the wikipedia article just uses `s1s2` etc. for the different values/rows.